packages/cubejs-api-gateway/src/gateway.ts

```diff
@@ -160,6 +160,8 @@ export interface ApiGatewayOptions {
   standalone: boolean;
   dataSourceStorage: any;
   refreshScheduler: any;
+  scheduledRefreshContexts?: any;
```

Comment:
Circular dependency for types. We can resolve it by moving `UserBackgroundContext` type to `api-gateway` package and reference it here and in `server-core/src/core/types.ts`.